pylint/typing.py

```diff
@@ -51,6 +51,7 @@ class ModuleDescriptionDict(TypedDict):
     isarg: bool
     basepath: str
     basename: str
+    isignored: bool
```

Comment:
Was initially hoping to have all ignored files reporting go through this, but that 
seems like it would be a larger refactor and out of scope.